### DISCUSSION TITLE
AVAST: false positive

### DISCUSSION THREAD

**Discussion by Eidonko** [jamulussoftware/jamulus#3247] [General]
Dear Sir / Madam,

I have recently found out about Jamulus -- first of all, my thanks for creating such useful software. I have installed it on my Ubuntu laptop with Jackd. This is to ask your help with a problem that a friend of mine, with whom I'd like to test Jamulus, has experienced. He's using Windows and the AVAST antivirus.  Regrettably Jamulus erroneously results positive on AVAST, thus my friend has not installed it. This is to kindly ask your suggestion: what would make more sense, contact AVAST to discuss the case? Or would you know of any workaround? Many thanks,

Kind regards,
Eidon

**Comment by rdica** [comment]
Hi @Eidonko have you looked at https://github.com/orgs/jamulussoftware/discussions/3175 ?
This is a discussion on this very same issue.
